.travis.yml

```diff
@@ -8,6 +8,16 @@ os:
   - linux
 #  - osx
 
+addons:
+  sauce_connect:
+    username: "robertknight"
+    access_key: "6775e8c2-68fb-4f12-9c12-2bf326d2eba3"
```

Comment:
I have included this key in plain text for now, when this lands on master it should have its own set of credentials added owned by the repository admin and added to .travis.yml using `travis encrypt`

Sauce Labs accounts are free for OSS projects.